app/controllers/posts_controller.rb

```diff
@@ -19,10 +19,13 @@ def create
     post = Post.new(post_params.merge(author_id: @current_user.id, comments_counter: 0, likes_counter: 0))
     if post.valid?
       post.save
-      redirect_to user_path(@current_user.id)
+      flash.notice = 'Post added successfuly.'
+      # redirect_to user_path(@current_user.id)
```

Comment:
- [ ] As requested by the previous reviewer kindly remove the commented code to make your code cleaner